s3config-test.yaml

```diff
@@ -19,6 +19,9 @@
 
 S3Config_Sections: [S3_SERVER_CONFIG, S3_AUTH_CONFIG, S3_MOTR_CONFIG, S3_THIRDPARTY_CONFIG]
 S3_SERVER_CONFIG:                                     # Section for S3 Server
+   S3_RANGED_READ_ENABLED: true                         # enable ranged reads
```

Comment:
Shouldn't the default be false since we are disabling this?

Reply:
It should. Let's see if S3server team prefers to keep it enabled in the branch or it could be disabled in the branch for all use cases.